extras/videoDrivers/SDL2/VideoSDL2.cpp

```diff
@@ -213,29 +213,27 @@ void VideoSDL2::PrintError(const std::string& msg) const
 
 void VideoSDL2::HandlePaste()
 {
-#ifdef _WIN32
-    if(!IsClipboardFormatAvailable(CF_UNICODETEXT))
+    if(!SDL_HasClipboardText())
         return;
 
-    OpenClipboard(nullptr);
-
-    HANDLE hData = GetClipboardData(CF_UNICODETEXT);
-    const wchar_t* pData = (const wchar_t*)GlobalLock(hData);
+    auto* text = SDL_GetClipboardText();
```

Comment:
Can you please put that into a `std::unique_ptr` with a deleter calling `SDL_free`? Might even make sense to use a type alias like `SDL_memory` or so.  
We should only use full RAII in new/changed code.

Reply:
Sure.

Reply:
I put it in the source file since it isn't needed in the header and didn't put the `SDLMemoryDeleter` in a `detail` namespace.